analyses/cell-type-scimilarity/scripts/02-run-scimilarity.py

```diff
@@ -155,6 +170,8 @@ def main() -> None:
       "scimilarity_celltype_annotation": predictions.values, 
       "min_dist": nn_stats["min_dist"]
     })
+    # add in ontology IDs
+    predictions_df = predictions_df.join(ontology_map.set_index('scimilarity_celltype_annotation'), on='scimilarity_celltype_annotation')
```

Comment:
I'm assuming if the index was already set you don't need to reset it here. I also added validation, which I think is correct? 
```suggestion
    predictions_df = predictions_df.join(ontology_map, on="scimilarity_celltype_annotation", validate="many_to_one")
```